code/builders/builder-vite/src/index.ts

```diff
@@ -85,14 +84,17 @@ export const build: ViteBuilder['build'] = async ({ options }) => {
   const previewDirOrigin = previewResolvedDir;
   const previewDirTarget = join(options.outputDir || '', `sb-preview`);
 
-  const previewFiles = fs.copy(previewDirOrigin, previewDirTarget, {
+  // TODO: `fsPromises.cp` is marked as experimental in Node 16-21. Ask in the PR whether we should
+  // use it anyway or stick to `fs-extra` for now.
```

Comment:
You'll notice this comment in several places in this PR. Since `cp` and `cpSync` are not experimental only from Node 22 onwards, should we:
1. Use them anyway?
2. Use `fs-extra` in these cases?
3. Use a combination of `fs.stat`, `fs.mkdir`, and `fs.open` instead of `cp`/`cpSync`?

Reply:
From what I understand, it was marked as experimental but did exist for a while in node versions before 22 right? 

i.e earlier versions do support it whether it is considered experimental or not

If that's true, I wonder if we should just use it anyway

Let's wait for someone else to review too though

Reply:
It was added in 16.7.0 and the API seems to have stayed rather consistent since then, so maybe it's fine to use it. The final call is up to the maintainers though. 🙂

Reply:
Thanks for the heads up. 🙏 It looks like it was experimental then finally stable based on that it would be extremely weird for them to remove/modify the "experimental" API in v18/v20, so I'm fine with keeping it.

Let's remove the comment(s).